LondonBritishLibrary/orient/BLorient608.xml

```diff
@@ -59,58 +160,75 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                         <width>337</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="12" cert="high"/>
-                                
+
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a fine character, between  <date notBefore="1716" notAfter="1721"/>.
-                                    
+
                                 </desc>
-                                
+
                                 <date notBefore="1716" notAfter="1721" resp="PRS10171Wright"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
-                        <additions>
-                            
-                            
-                        </additions>
+
+                        </handDesc>
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<locus target="2r"/>
+<desc type="ScribalSignature">The name of the scribe was <persName role="scribe" ref="PRS11360Adara">ʾAdarā Krǝstos</persName>.</desc>
+</item>
+
+<item xml:id="a2">
+<locus from="1r" to="1v"/>
+<desc type="Supplication"/>
+<q xml:lang="gez">በረከቱ፡ ወበረከተ፡ ኵሎሙ፡ ቅዱሳን፡ መላእክት፡ የሀሉ፡ ምስለ፡ ፍቁሩ፡ ንጉሥነ፡ ዳዊት። ወምስለ፡ ነፍሰ፡ አቡሁ፡ ንጉሥነ፡ ኢያሱ። ወምስለ፡ እሙ፡
+ቅድስተ፡ ክርስቶስ፡ ለዓለመ፡ ዓለም፡ አሜን።</q>
+<note>The note states that the manuscript was written for emperor <persName role="donor" ref="PRS3430DawitII">Dāwit</persName>.
+Mentioned are also his parents: emperor <persName ref="PRS5616IyasuI">ʾIyāsu</persName> and <persName ref="PRS13407QeddestaKrestos">Qǝddǝsta Krǝstos</persName>.</note>
+</item>
```

Comment:
I am not sure that this can be classified as an addition, if it is written in the same hand as the main text. What do you think @DenisNosnitsin1970 ?